packages/api/AlvTimeWebApi/Controllers/Admin/ProjectController.cs

```diff
@@ -9,20 +9,18 @@ namespace AlvTimeWebApi.Controllers.Admin
     [ApiController]
     public class ProjectController : Controller
     {
-        private readonly IProjectStorage _storage;
-        private readonly ProjectCreator _creator;
+        private readonly ProjectService _projectService;
 
-        public ProjectController(IProjectStorage storage, ProjectCreator creator)
+        public ProjectController(ProjectService projectService)
         {
-            _storage = storage;
-            _creator = creator;
+            _projectService = projectService;
         }
 
         [HttpGet("Projects")]
         [AuthorizeAdmin]
         public ActionResult<IEnumerable<ProjectResponseDto>> FetchProjects()
         {
-            return Ok(_storage.GetProjects(new ProjectQuerySearch()));
+            return Ok(_projectService.GetProjects(new ProjectQuerySearch()));
```

Comment:
Ok()

Reply:
Skjønner ikke denne kommentaren. Skal jeg fjerne Ok() på samme måte som din tidligere kommentar (i CustomerController), eller skrive for eksempel

`_projectService.GetProjects(new ProjectQuerySearch());
return Ok();`

Reply:
Samme som over, fjern Ok så det blir `return _projectService.GetProjects(new ProjectQuerySearch());`